.github/workflows/tpu-testing.yml

```
@@ -127,6 +127,5 @@ jobs:
         token: ${{ secrets.CODECOV_TOKEN }}
         file: coverage
         flags: tpu,pytest
-        env_vars: ${{ runner.os }},python
         name: TPU-coverage
-        fail_ci_if_error: true
+        fail_ci_if_error: false
```

```suggestion
        fail_ci_if_error: true
```
we shall keep this one for sure as with no report the coverage will be lower and it triggers codecov failed and with having TPU test green you would not suspect just missing report but rather some code issue

---

ok